### DISCUSSION TITLE
S7comm Zeek Plugin

### DISCUSSION THREAD

**Discussion by YannickZ5** [Security-Onion-Solutions/securityonion#8501] [Unsupported Versions]
Hello everyone,

we are running Security Onion v2.3.130 in an ICS environment as a standalone and air-gapped installation. Is there a possibility to run this https://github.com/amzn/zeek-plugin-s7comm Plugin within Security Onion? I've tried the steps from the offical documentation but unfortunately i get the following error message:

`error in /opt/zeek/share/zeek/policy/custom/s7comm/./main.zeek, line 59: unknown indentifier Analyzer::ANALYZER_S7COMM, at or near "Analyzer::ANALYZER_S7COMM`

Thank you for your help!

**Comment by dougburks** [comment]
Looks like that plugin requires a compiled component in addition to the Zeek scripts themselves.

**Reply by YannickZ5** [reply]
Hey, thanks for your reply.

So normally I would use the zkg command line tool. Can you tell me if there is an offical supported way to do it in Security Onion?
